src/utils/plist.ts

```diff
@@ -0,0 +1,18 @@
+import { readFileSync } from 'fs'
+import { parse } from 'plist'
+import log from 'electron-log/main'
+
+export function getExecutableNameFromPlist(plistPath: string) {
+  const plistContent = readFileSync(plistPath, 'utf-8')
+  const plistData = parse(plistContent) as Record<string, unknown>
+
+  if (typeof plistData.CFBundleExecutable === 'string') {
+    return plistData.CFBundleExecutable
```

Comment:
[CFBundleExecutable](https://developer.apple.com/documentation/bundleresources/information-property-list/cfbundleexecutable) is the default location for the executable name